docker-compose.azure.yml

```diff
@@ -6,6 +6,7 @@ services:
       args:
         bundleWithout: 'development'
         BASE_RUBY_IMAGE_WITH_GEMS_AND_NODE_MODULES: ${gemsNodeModulesImageName:-apply-for-teacher-training-gems-node-modules}
+        SHA: ${dockerHubImageTag}
```

Comment:
Maybe something like `DEPLOYED_VERSION` instead of `SHA` would be clearer?

Reply:
began with `VERSION`, I'll change this to `VERSION` but realised OKComputer used `SHA` and changed all occurances,
will change this entry to `VERSION`